wrappers/javascript/packages/anoncreds-nodejs/package.json

```diff
@@ -27,7 +27,7 @@
     "install": "node scripts/install.js"
   },
   "dependencies": {
-    "@2060.io/ffi-napi": "4.0.8",
+    "@2060.io/ffi-napi": "4.0.9",
```

Comment:
Maybe we should use caret versions so we can publish fixes to the ffi-napi in the future without having to make new releases for the wrappers